### DISCUSSION TITLE
x-text with a a template string is it possible?

### DISCUSSION THREAD

**Discussion by mbokil** [alpinejs/alpine#4243] [1. Help] [answered]
I was trying to use a template string in the x-text directive but it doesn't appear to work. I'm pretty certain it works in Vue 3. Is it possible to do this in Alpine? 
```
<span x-text="`${tx.date} ${tx.dayOfWeek}`">
```

Instead I went back to ES5 string concatenation and it worked
`<span x-text="tx.date+' '+tx.dayOfWeek"></span>`

**Comment by SimoTod** [comment, marked as answer]
It should work in Alpine too: https://codepen.io/SimoTod/pen/qBGRRpe

**Reply by mbokil** [reply]
Thanks I appreciate the help. Interestingly looks like the JSP engine I am using interprets the template strings as JSP expressions. So I have to escape them `\${subject}`
